Description: add /usr/games to path
 For unknown reasons the $PATH is not set right in Kubuntu.
 Until this bug is fixed properly add a workaround to just
 add /usr/games to startkde
Author: Scarlett Clark <sgclark@kubuntu.org>
Last-Update: 2014-11-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/startkde/startkde.cmake
+++ b/startkde/startkde.cmake
@@ -3,6 +3,7 @@
 #  DEFAULT KDE STARTUP SCRIPT ( @PROJECT_VERSION@ )
 #
 
+export PATH=$PATH:usr/games
 export QML2_IMPORT_PATH=/usr/lib/@DEB_HOST_MULTIARCH@/qt5/qml/
 QTPATHS=/usr/lib/@DEB_HOST_MULTIARCH@/qt5/bin/qtpaths
 
